### ISSUE TITLE
Generate Protein - Ligand System

### ISSUE THREAD

**Issue by entropybit** [openmm/openmm-ml#81]
Hi, 

I am wondering what would be a clean way to create a protein ligand system where the ligand is parametrized with 
one of the MLFFs ?  

In the example it is shown how to create a mixed system but this requires already having a topology + system
parametrized without the MLFF.
Currently I am trying to do this by just creating a mixed system using one of the small molecule FFs together with 
e.g. amber. So I get a system where my Protein is parametrized with amber and the Ligand with the small molecule FF. 
I can then use this system and topology together with a list of atom ids for the ligand to use the mixed system functionality from 
openmm-ml but this seems very messy. 

Is there a better way, or am I maybe doing something horribly wrong in this approach to start with ?

**Comment by peastman** [comment]
In the mixed system, the internal energy of the ligand is computed with the ML potential, but the interaction between the ligand and everything else is computed with the conventional force field.  That's why you need to parameterize everything, including the ligand, with the force field.  Or am I misunderstanding what you're asking?

**Comment by entropybit** [comment]
@peastman No you are right, I just did not think about this....
Does it even make sense to do this ? 

It would be better to also have the MLFF for the interactions between Ligand and Protein too. But I can see that this is not easily done. Unless the ML part is used to generate parameters in the language of an classical FF, like Espaloma does. 
Mixing rules can not really be applied to the MLFF part right ?

**Comment by peastman** [comment]
Right.  Different ML potentials work in different ways, but most often they just throw all the atom positions into a neural network and produce an energy for each atom at the end.  There's no way to separate that energy into different components, such as the part that came from the interaction between two molecules.
